librz/bin/p/bin_elf.c

```diff
@@ -11,20 +11,89 @@ static void destroy(RzBinFile *bf) {
 
 static void headers32(RzBinFile *bf) {
 #define p bf->rbin->cb_printf
-	p("0x00000000  ELF MAGIC   0x%08x\n", rz_buf_read_le32_at(bf->buf, 0));
-	p("0x00000010  Type        0x%04x\n", rz_buf_read_le16_at(bf->buf, 0x10));
-	p("0x00000012  Machine     0x%04x\n", rz_buf_read_le16_at(bf->buf, 0x12));
-	p("0x00000014  Version     0x%08x\n", rz_buf_read_le32_at(bf->buf, 0x14));
-	p("0x00000018  Entrypoint  0x%08x\n", rz_buf_read_le32_at(bf->buf, 0x18));
-	p("0x0000001c  PhOff       0x%08x\n", rz_buf_read_le32_at(bf->buf, 0x1c));
-	p("0x00000020  ShOff       0x%08x\n", rz_buf_read_le32_at(bf->buf, 0x20));
-	p("0x00000024  Flags       0x%08x\n", rz_buf_read_le32_at(bf->buf, 0x24));
-	p("0x00000028  EhSize      %d\n", rz_buf_read_le16_at(bf->buf, 0x28));
-	p("0x0000002a  PhentSize   %d\n", rz_buf_read_le16_at(bf->buf, 0x2a));
-	p("0x0000002c  PhNum       %d\n", rz_buf_read_le16_at(bf->buf, 0x2c));
-	p("0x0000002e  ShentSize   %d\n", rz_buf_read_le16_at(bf->buf, 0x2e));
-	p("0x00000030  ShNum       %d\n", rz_buf_read_le16_at(bf->buf, 0x30));
-	p("0x00000032  ShrStrndx   %d\n", rz_buf_read_le16_at(bf->buf, 0x32));
+	ut32 magic;
```

Comment:
Here as well - could be a macro

Reply:
I am refactoring this file, so this is temporary.